LBH.AdultSocialCare.Api/V1/Extensions/IQueryableExtensions.cs

```diff
@@ -15,5 +15,15 @@ public static IQueryable<T> GetPage<T>(this IQueryable<T> list, int pageNumber,
                 .Skip((pageNumber - 1) * pageSize)
                 .Take(pageSize);
         }
+
+        public static IQueryable<T> TrackChanges<T>(this IQueryable<T> list, bool trackChanges) where T : class
```

Comment:
Consider using default value, true, I think

Reply:
Good idea. Though default to false. Only set to true manually if you are going to update the entity

Reply:
Have some feeling that we will update more than read, but it's just vague feeling, so up to you :)